src/libcore/ops/function.rs

```diff
@@ -8,27 +8,38 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-/// A version of the call operator that takes an immutable receiver.
+/// The version of the call operator that takes an immutable receiver.
 ///
-/// Closures only taking immutable references to captured variables
-/// automatically implement this trait, which allows them to be invoked.
-/// For mutably referenced captures, see [`FnMut`], and for consuming the
-/// capture, see [`FnOnce`].
+/// Instances of `Fn` can be called repeatedly without mutating state.
 ///
-/// You can use the `Fn` traits when you want to accept a closure as a
-/// parameter. Since both [`FnMut`] and [`FnOnce`] are supertraits of `Fn`, any
-/// instance of `Fn` can be used where a [`FnMut`] or [`FnOnce`] is expected.
+/// *This trait (`Fn`) is not to be confused with [function pointers] (`fn`).*
+///
+/// `Fn` is implemented automatically by closures which only take immutable
+/// references to captured variables or don't capture anything at all, as well
+/// as (safe) [function pointers][] (with some caveats, see their documentation
```

Comment:
I checked, and I do need them (because of hoedown, I wouldn't in CommonMark :unamused:)